commands/moderation/verbal.js

```diff
@@ -33,11 +33,13 @@ module.exports = {
         return msg.reply(`${e.name}: ${e.message}`);
       }
 
-      // Register call in the Audit-log channel
+      // Register call in the audit-log channel
       auditLog(msg, offendingUser, verbalReason);
 
-      // Give SU, Mod, Admin feedback on their call
-      msg.channel.send(`${msg.author} just verballed ${offendingUser}`);
+      // Give Mod or Admin feedback on their call
+      msg.channel.send({
+        content: `${msg.author} just verballed ${offendingUser}`,
+      });
```

Comment:
As I mentioned in Mike's last PR, just a heads up that we don't need to send an options object with `content` if all we're sending is a string. Any non-empty string is fine if we're not sending other content like embeds.